<launch>
    <node pkg="car_master" type="imu_node" name="imu_node" respawn="true" respawn_delay="1" output="screen" >
        <param name="port_name" value="/dev/ttyS0" />
        <param name="baud" value="115200" />
        <param name="frame_id" value="Imu" />
        <param name="imu_topic" value="imu_data" />
    </node>
</launch>